chromatic/rainbows/actions/shift.py

```diff
@@ -12,8 +12,8 @@ def shift(self, velocity=5*u.km/u.s):
     
     new = self._create_copy()
     
-    c = 3e5 * u.km / u.s #speed of light in km/s
-    new_wavelength = new.wavelength * ( 1 / (1 - velocity/c) )
+    lightspeed = c.c.to('km/s') #speed of light in km/s
```

Comment:
The `chromatic/imports.py` already imports the astropy constants module with `import astropy.constants as con`, so I think you can just say `lightspeed = con.c` here.